src/pages/components/AdditionalResources/index.js

```diff
@@ -11,47 +11,46 @@ const AdditionalResources = () => {
     <Box className="containerSec">
       <Box className="section">
         <Box className="titleBox">
-          <h2>Additional Resources</h2>
+          <h2>Recursos adicionales</h2>
```

Comment:
>Recursos Adicionales (Title Case)

Reply:
@xavier506 not necessary in Spanish to add title cases